packages/demo/src/demo-building-blocs/ExampleLayout.tsx

```diff
@@ -43,32 +46,36 @@ export const ExampleLayout: React.FunctionComponent<ExampleLayoutProps> = ({
             />
             <TabContent>
                 <TabPaneConnected id="implementation" groupId="page">
-                    {isShowingCode && <Content code={code} examples={examples} />}
+                    {isShowingCode && <Content id={id} code={code} examples={examples} />}
                 </TabPaneConnected>
                 <GuidelinesTab id={id} />
             </TabContent>
         </div>
     );
 };
-const Content: React.FunctionComponent<Pick<ExampleLayoutProps, 'code' | 'examples'>> = ({code, examples}) => (
+const Content: React.FunctionComponent<Pick<ExampleLayoutProps, 'code' | 'examples' | 'id'>> = ({
+    code,
+    examples,
+    id,
+}) => (
     <>
         <div className="example-layout__main-code example-layout__section">
             <Sandbox id="main-code">{code}</Sandbox>
         </div>
         <div className="example-layout__props example-layout__section">
-            <h4 className="h2 mb5">Props</h4>
-            <div>Insert table props here</div>
+            <h2 className="mb1">Props</h2>
```

Comment:
I used h4 to be semantically correct since the h2 (e.g., inputs) is the section and the h3 the title (e.g., buttons). I think we should use semantically correct tags and override the style with classes

Reply:
oh right ok, I will revert back, I was wondering why it was that way